docker-compose/backoffice-docker-compose.yml

```diff
@@ -26,12 +26,8 @@ services:
     image: redis:3.2-alpine
 
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:5.4.1
+    image: kuzzleio/elasticsearch:5.4.1
     environment:
       - cluster.name=kuzzle
-      # disable xpack
-      - xpack.security.enabled=false
-      - xpack.monitoring.enabled=false
-      - xpack.graph.enabled=false
-      - xpack.watcher.enabled=false
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
```

Comment:
Do we need to specify this value, as `512m` is already the default value in the container image (https://github.com/kuzzleio/kuzzle-containers/blob/master/elasticsearch/Dockerfile#L6) ?

Reply:
the idea is to autodocument this param to let users easily change it according to their needs.